content.md

```diff
@@ -88,6 +87,13 @@ This list is also available online at: https://resources.osucord.moe/
 - [Ronnia](https://ronnia.me/) - Twitch/osu! bot that sends beatmap requests from Twitch chat to the streamer's in-game messages
 - [JKPS](https://github.com/Tonetfal/JKPS) - Keyboard overlay featuring key press visualisation and a keys per second meter
 ---
+### Discord Bots
+- [owo](http://owo-bot.xyz/) - A general-purpose discord bot to show of your profile/recent plays, get map recommendations and compare yourself to others
+- [BathBot](https://github.com/MaxOhn/Bathbot) - A feature-rich discord bot with functionality all around osu!
+- [Yuna](https://discord.com/oauth2/authorize?client_id=832597585923014676&scope=bot&permissions=60480) - The official [o!rdr](https://ordr.issou.best/) bot lets you render your replays through discord chat
+- [AxerBot](https://github.com/Hiviexd/AxerBot) - A general-purpose and powerful Discord bot with osu! features related to mapping & modding
+- [osu! Miss Analyzer](https://github.com/ThereGoesMySanity/osuMissAnalyzer) - A program to analyze misses in an osu! replay (can be used locally or through discord)
```

Comment:
```suggestion
- [osu! Miss Analyzer](https://github.com/ThereGoesMySanity/osuMissAnalyzer) - A tool to analyze misses in an osu! replay (has a discord bot and program to run locally)
```

kinda having second thoughts on moving this and thinking we might just want to put it in both instead, with links pointing to either the main/bot readme